internal/ensure/requiredversion.go

```diff
@@ -147,6 +147,11 @@ func goModVersion(dir string) (moduleVersion string, moduleDir string, err error
 		return "", "", errors.Join(errs...)
 	}
 
+	// Shouldn't happen but does: see https://github.com/golang/go/issues/65816
+	if pkg.Module == nil {
+		return "", "", fmt.Errorf("no module information found for package %q", pkg.PkgPath)
```

Comment:
Interesting this shows as un-tested...

Reply:
This must be linked to the fact that we don't have this issue in CI but I do on my machine. I can't really understand why because I mapped exactly the go version on my machine with the CI :shrug:

Reply:
I think the difference is on your local machine you probably did `go install github.com/DataDog/orchestrion` so it can be found under `GOPATH`, but it's not true on CI... (so `packages.Load` actually returns an error).

Reply:
Wholly resetting the GOPATH does not change a thing :/